src/cmd/backup.go

```diff
@@ -16,6 +16,14 @@ import (
 // Backup stores original apps packages, extracts them and preprocesses
 // extracted apps' assets
 func Backup(spicetifyVersion string) {
+	if isAppX {
+		utils.PrintInfo(`You are using Spotify Windows Store version, which is only partly supported.
+Stop using Spicetify with Windows Store version unless you absolutely CANNOT install normal Spotify from installer.
+Modded Spotify cannot be launched using original Shortcut/Start menu tile. To correctly launch Spotify with modification, please make a desktop shortcut that execute "spicetify auto". After that, you can change its icon, pin to start menu or put in startup folder.`)
+		if !ReadAnswer("Continue backing up anyway? [y/N]: ", false, true) {
+			os.Exit(1)
+		}
+	}
```

Comment:
You added this in other pr and I think it can make an issue while merging this one but idk

Reply:
Yeah I froze for a bit when I found it lmao, I'll merge it in this one instead since it comes with the whole package.